### PR TITLE
Build the basic release docker image

### PR DESCRIPTION
**Pull request by silathdiir**
### Summary

1. Adds a shell script `release/release.sh` and Dockerfile to build the `basic` docker image for release.
2. Updates README.
3. Upgrades `rdkafka` from `0.25.0` to `0.26.0` for the below issue when building  for GNU.
```
error[E0308]: mismatched types
   --> /cargo/registry/src/github.com-1ecc6299db9ec823/rdkafka-0.25.0/src/config.rs:153:17
    |
153 |                 buf.as_mut_ptr() as *mut i8,
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `u8`, found `i8`
    |
    = note: expected raw pointer `*mut u8`
               found raw pointer `*mut i8`
```

### CODE REVIEW

File: release/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:20.04
+
+COPY config /config
+COPY target/aarch64-unknown-linux-gnu/debug/matchengine /usr/bin/
```
Comment (lispc) [inline release/Dockerfile:4]: why aarch64 rather than x64/amd64?

aarch64 is arm64/armv8

Comment (silathdiir) [reply release/Dockerfile:4]: Needs to be fixed to `x86_64-unknown-linux-gnu`.

File: README.md [hunk 2]
```diff
@@ -61,6 +61,22 @@ $ cd $DingirExchangeDir/examples/js ; npm i
 $ npx ts-node trade.ts 
 ```
 
+## Release
+
+We uses [cross](https://github.com/rust-embedded/cross) to generate release builds for Linux Distributions.
+For example, you could generate a static release build via the below command.
+
+```
+cross build --target aarch64-unknown-linux-gnu --release
```
Comment (lispc) [inline README.md:70]: update aarch64 here

**Review by lispc — DISMISSED** [review]

**Comment by silathdiir** [comment]
Since the issue https://github.com/rust-lang/cargo/issues/7878 is opened, updates `RUSTFLAGS` as a build parameter.

**Review by lispc — APPROVED** [review]

**Pull request merged**